src/composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "autoload": {
+        "psr-4": {
+            "BiscuitApp\\": "."
+        }
+    },
```

Comment:
put this bit on the end (after `require-dev`) just for tidiness so the file starts with `name` `description` etc

Reply:
resolved